test_gast_to_code.py

```diff
@@ -38,13 +38,16 @@
             "type": "root",
             "body": [
                 {
-                    "type": "logStatement",
+                    "type": "func",
```

Comment:
Can we add a test for custom functions as well as logStatements?

Reply:
I just added a test file for custom functions that I am about to push